sample/FOSSASIA11/tracks

```diff
@@ -12,19 +12,7 @@
             },
             {
                 "id": 2,
-                "title": "Mobile Solutions"
```

Comment:
@iamareebjamal Actually it is not tracks i have removed. It is sessions for this tracks. That don't exists in sessions file and was giving error while testing.

And this sample is not complete according to http://2011.fossasia.org